src/guake/terminal.py

```diff
@@ -74,8 +74,8 @@
      r"^[-a-zA-Z0-9\/\_\.\ ]+\.?[a-zA-Z0-9]+\:[0-9]+",
      r"^(.*)\:([0-9]+)"),
     ("line containing '/home/' absolute path",
-     r"(/home/[a-zA-Z0-9_\-\./]+)(:[0-9]+)?",
-     r"(/home/[a-zA-Z0-9_\-\./]+):?([0-9]+)?")
+     r"(/home/[-a-zA-Z0-9_\./]+)(:[0-9]+)?",
```

Comment:
I checked in pure python re, `\-` works perfectly  
It seems that regexp doesn't work in vte.Terminal.
I don't know what happens after `vte.Terminal.match_add(match)`